lightning/src/util/events.rs

```diff
@@ -160,7 +160,6 @@ impl Writeable for Event {
 				write_tlv_fields!(writer, {
 					(0, payment_preimage, required),
 				});
-				payment_preimage.write(writer)?;
```

Comment:
Darn! If anyone wants to post-mortem, this seems to be the commit where this occurred: https://github.com/rust-bitcoin/rust-lightning/pull/935/commits/263bf9e7951240096c8fdedb3c90279e53bd688b#diff-3b5c370b404e2f5a8f3c35093b97406f149a9340c177c05252574083d68df0daR160-R163 At least serialization was a bit of a one-off big PR.

Reply:
Yea, good thing we have fuzzers (when I remember to check their output...).

Reply:
AFAICT, it was introduced by https://github.com/rust-bitcoin/rust-lightning/pull/935 ?

Reply:
Yes, I believe so.